bio_hansel/subtyper.py

```diff
@@ -437,6 +438,37 @@ def absent_downstream_subtypes(subtype: str, subtypes: pd.Series, scheme_subtype
     return absentees if len(absentees) > 0 else None
 
 
+def missing_nests(st: Subtype, subtype: str, df_positive: pd.DataFrame) -> Subtype:
+    """Find nested subtypes that are missing from the final subtype call
+
+    Args:
+        subtype: Final subtype result
+        positive_subtypes: List of unique positive subtypes found
+    
+    Returns:
+        List of missing hierarchical subtypes or `None` if there are no missing nested hierarchical subtypes.
+    """
+
+    pos_subtypes_str = [x for x in df_positive.subtype.unique()]
+    broken_subs = re.compile(r'^\d(\d*\.+\d+)*$')
+    primary_subtypes = []
+    for value in range(len(subtype)+1):
```

Comment:
Using `i` instead of `value` might help the code look a little more idiomatic so it would look similar to a `C/C++/Java` for index loop.